style.css

```diff
@@ -196,6 +230,10 @@ footer {
   position: relative;
 }
 
+article img {
+  width: 100%;

```

Comment:
- [ ] As much as possible prefer using Bootstrap classes whenever possible.
Width 100% can be set using a Bootstrap class, search [in the documentation here](https://getbootstrap.com/docs/4.1/getting-started/introduction/) for width.

Other styles that can be set with Boostrap in your project:
- display
- position
- color black or white
- font weight
- flex justify and align

kindly review your stylesheets and use the use the Boostrap classes in the HTML whenever the substitution is possible. That will help you remember all the areas that Bootstrap can cover and be useful for quickly styling your next projects.